packages/calling/src/CallingClient/calling/call.test.ts

```diff
@@ -712,6 +712,33 @@ describe('Call Tests', () => {
       {file: 'call', method: 'updateMedia'}
     );
   });
+
+  describe('#addSessionConnection', () => {
+    let call;
+
+    beforeEach(async () => {
+      call = callManager.createCall(dest, CallDirection.INBOUND, deviceId, mockLineId);
+    });
+
+    it('should add a session connection', () => {});
```

Comment:
this is an empty test?

Reply:
Sorry. I started it there but somehow missed to remove it. Removed it now.